src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -200,7 +203,7 @@ internal static WorkerInitResponse WorkerInitRequestHandler(WorkerInitRequest re
                 WorkerVersion = WorkerInformation.Instance.WorkerVersion,
                 WorkerMetadata = new WorkerMetadata
                 {
-                    RuntimeName = RuntimeInformation.FrameworkDescription,
+                    RuntimeName = _frameworkDescriptionRegex.Match(RuntimeInformation.FrameworkDescription).Value.Trim() ?? RuntimeInformation.FrameworkDescription,
```

Comment:
Wonder the regex can be updated to handle the space split so you can remove the Trim call here.

Reply:
Ah good point, can definitely do that!